starfish/image/_filter/bandpass.py

```diff
@@ -25,15 +25,19 @@ def __init__(
         llong : int
             filter frequencies above this odd integer value
         threshold : float
```

Comment:
didn't notice this in the first round, but this (and `truncate`) are declared as `floats` in the doc, but as `Number` in the params.